base_layer/core/src/mining/miner.rs

```diff
@@ -77,7 +86,9 @@ impl Miner {
         threads: usize,
     ) -> Miner
     {
-        let (utxo_sender, _): (Sender<UnblindedOutput>, Receiver<UnblindedOutput>) = mpsc::channel(1);
+        let (utxo_sender, _): (mpscSender<UnblindedOutput>, mpscReceiver<UnblindedOutput>) = mpsc::channel(1);
+        let (miner_instruction_events, _): (syncSender<MinerInstruction>, syncReceiver<MinerInstruction>) =
+            broadcast::channel(10);
```

Comment:
Is there a reason we want to que more than 1 instruction. Dont we just want the last instruction?

Reply:
Strictly speaking not necessary, but this is just to make sure we do not drop any commands from the CLI, a bit of resilience